variables.tf

```
@@ -83,6 +89,32 @@ variable "default_branch" {
   description = "Name of the default branch for the GitHub repository"
 }
 
+variable "default_branch_protection" {
+  type = object({
+    enforce_admins         = optional(bool, false)
+    require_signed_commits = optional(bool)
+
+    required_checks = optional(object({
+      strict   = optional(bool)
+      contexts = optional(list(string))
+    }))
+
+    restrict_pushes = optional(object({
+      blocks_creations = optional(bool)
+      push_allowances  = optional(list(string))
+    }))
+
+    required_reviews = optional(object({
+      dismiss_stale_reviews           = optional(bool, true)
+      dismissal_restrictions          = optional(list(string))
+      required_approving_review_count = optional(number, 2)
```

```suggestion
      required_approving_review_count = optional(number, 2)
```

Nobody is doing this, shall we set the default to 1? I think this checkov rule is already globally excluded

---

This may be the case, but the CIS Software Supply Chain Security guidelines control 1.1.3 says "Ensure any change to code receives approval of two strongly authenticated users." By doing this we also skip the exclude since we'd be following the best practice/recommendation.